runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -74,6 +74,9 @@
 	DIARY_QUEST_WIDGET_TITLE(WidgetID.DIARY_QUEST_GROUP_ID, WidgetID.Diary.DIARY_TITLE),
 	DIARY_QUEST_WIDGET_TEXT(WidgetID.DIARY_QUEST_GROUP_ID, WidgetID.Diary.DIARY_TEXT),
 
+	PEST_CONTROL_BOAT_INFO(WidgetID.PEST_CONTROL_BOAT_GROUP_ID, 2),
```

Comment:
A `PestControl` child class already exists, we may as well define this child ID there.

Reply:
The only reason I didn't want to put it in there is that it is a separate widget group entirely, and it would collide with the `KNIGHT_INFO_CONTAINER` that already exists in it.